model/host/host.go

```diff
@@ -318,11 +318,11 @@ func (h *Host) MarkAsProvisioned() error {
 func (host *Host) ClearRunningTask(prevTaskId string, finishTime time.Time) error {
 	host.LastTaskCompleted = prevTaskId
 	host.LastTaskCompletedTime = finishTime
-	host.RunningTask = ""
 	event.LogHostRunningTaskCleared(host.Id, prevTaskId)
```

Comment:
This line should probably go below the update now

Reply:
done